src/commands/queryHistory/applyQuery.ts

```diff
@@ -12,6 +12,7 @@ export const applyQuery = async (query: IQuery) => {
         return;
     }
     const activeTextEditor = vscode.window.activeTextEditor;
+    activeTextEditor?.document.save();
```

Comment:
Shouldn't we have it after if statement? This will ensure other files are not getting saved randomly